docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -219,27 +220,25 @@ First download the latest GDAL release version and untar the archive::
 
     $ wget https://download.osgeo.org/gdal/X.Y.Z/gdal-X.Y.Z.tar.gz
     $ tar xzf gdal-X.Y.Z.tar.gz
-    $ cd gdal-X.Y.Z
 
-Configure, make and install::
+For GDAL 3.6.x and greater, releases only support builds using ``CMake``. To
+build with ``CMake`` create a ``build`` folder in the GDAL directory, and step
+into it::
 
-    $ ./configure
-    $ make # Go get some coffee, this takes a while.
-    $ sudo make install
-    $ cd ..
+    $ cd gdal-X.Y.Z
+    $ mkdir build
+    $ cd build
 
-.. note::
+Finally, configure, make and install GDAL::
 
-    Because GeoDjango has its own Python interface, the preceding instructions
-    do not build GDAL's own Python bindings. The bindings may be built by
-    adding the ``--with-python`` flag when running ``configure``. See
-    `GDAL/OGR In Python`__ for more information on GDAL's bindings.
```

Comment:
This note was outdated, see [GDAL docs](https://gdal.org/development/building_from_source.html#cmdoption-arg-BUILD_PYTHON_BINDINGS-BOOL).